components/Admin/SignersTable.tsx

```diff
@@ -5,50 +5,57 @@ import {
   IconClose,
   IconEdit,
   isAddrEqual,
-  StatusIcon,
   Wallet,
   WALLET_PROPTYPE
 } from '@glif/react-components'
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
 
-const IconsWrapper = styled.div`
-  display: flex;
-  align-items: center;
+export const ButtonsTD = styled.td`
+  color: var(--gray-medium);
+  text-align: right;
 
-  > span {
-    display: block;
-    cursor: pointer;
-    line-height: 0;
-    margin-left: 0.5em;
-    transition: transform 0.1s ease-out;
-    &:hover {
-      transform: scale(1.2);
+  > div {
+    display: flex;
+    align-items: center;
+    justify-content: end;
+    gap: var(--space-s);
+
+    > span {
+      cursor: pointer;
+      transition: transform 0.1s ease-out;
+
+      &:hover {
+        transform: scale(1.2);
+      }
+
+      svg {
+        vertical-align: middle;
+      }
     }
   }
 `
 
 const SignerRow = ({
   signer,
-  userIsSigner,
+  isCurrentUser,
   onChange,
   onRemove
 }: SignerRowProps) => {
   return (
     <tr>
-      <td>
-        {userIsSigner && <StatusIcon color='purple' margin='4px 0 0 1em' />}
-      </td>
       <td>
         <AddressLink
           address={signer.robust || signer.id}
           shouldTruncate={false}
           hideCopyText={false}
         />
       </td>
-      <td>
-        {!userIsSigner && (
-          <IconsWrapper>
+      <ButtonsTD>
```

Comment:
@navFooh i think a user should be able to propose a change or removal of their own signer from a multisig

Reply:
@navFooh 
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/12353734/184045676-8d8d43e7-ae7a-4501-a468-4b02ee96cfd9.png">